subgraphs/erc721-holders/src/mappings/collection.ts

```diff
@@ -60,6 +62,16 @@ export function handleTransfer(event: TransferEvent): void {
     );
   }
 
+  // ERC721 collection created via registry list (name/symbol field initialized null)
```

Comment:
condition for updating a collection entity already created but without name/symbol (should not run for the default version of the subgraph without a token list)